Lib/fontTools/feaLib/parser.py

```diff
@@ -787,9 +792,11 @@ def parse_table_(self):
 
     def parse_table_GDEF_(self, table):
         statements = table.statements
-        while self.next_token_ != "}":
-            self.advance_lexer_()
-            if self.is_cur_keyword_("Attach"):
+        while self.next_token_ != "}" or (not self.ignore_comments and len(self.cur_comments_)):
```

Comment:
OK Let's do that then. In answer to @anthrotype, yes the number of fixed position parameters to any function is, well, fixed. But you can have any number of named parameters. There is nothing special about next() in that respect.